### ISSUE TITLE
The nvda when executing the reading continues in a webpage repeats some contents

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9361]
for some time when reading pages continuously with nvda + I realize that nvda during playback repeats some lines, or even skips a few lines.
This occurs on some pages, and I believe I have found an example to inform.
However the page is in Portuguese Brazil, I know the difficulty in identifying this problem, but if you can identify the possible cause would be of great value.
### Steps to reproduce:
open the page:
https://www.c1d.com.br/single-post/2017/11/02/How-to-reiniciar-a-sua-mente-and-praticar-bons-h%C3%A-bito-Kau-Mascarenhas
and if you are using the notebook layout press nvda + a to read the page continuously.
### Actual behavior:
nvda during reading continues with nvda + a repeats some text snippets, or depending on you can even skip a few lines of text.
### Expected behavior:
the independence of page programming should be able to read the text continuously without skipping or repeating strings of texts.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
alfa 16885
#### Windows version:
17763.348
#### Name and version of other software in use when reproducing the issue:
chrome 72
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by Adriani90** [comment]
Is the cursor jumping backwards on this page when navigating with down arrow through the page?

**Comment by Brian1Gaff** [comment]
From other times I've seen this, no its not, its only in say all that the 
effects seem to occur, but then again how quick can you cursor, you have to 
allow for the line to finish reading.
 I thought maybe it was some kind of on the fly javascript rendering going 
on behind the scenes and allied it to the cases, usually when a browser is 
loaded of non automatic reading and the focus not in the page at all quite 
often.

Its a problem that has been around for years.

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by fernando-jose-silva** [comment]
I noticed a similar problem now when downloading in chrome.
Make a download that is time consuming in chrome.
Press ctrl + j.
On the chrome downloads page, when the download is performed, walk with the arrows above and below.
When navigating through downloads, nvda skips a few lines, or repeats a few lines.
The focus becomes erratic as if the page was updating and inserting lines.
I noticed this on chrome 73.
using nvda alfa 16976 and windows 10 17763.379
log:
[downloadchrome.txt](https://github.com/nvaccess/nvda/files/3023641/downloadchrome.txt)
